ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env bash
+
+source "${USHgfs}/preamble.sh"
+
+#-------------------------------------------------------------------------------------------------
+# Script to regrid surface increment from GSI grid 
+# to fv3 tiles. 
+# Clara Draper, Dec 2024
+#-------------------------------------------------------------------------------------------------
+
+# CSD: temporary on hera, until scrip files added to fix dir. 
+TMP_FIX_FILES=/scratch2/BMC/gsienkf/Clara.Draper/gw_new_fix_files/
+
+export PGMOUT=${PGMOUT:-${pgmout:-'&1'}}
+export PGMERR=${PGMERR:-${pgmerr:-'&2'}}
+export REDOUT=${REDOUT:-'1>'}
+export REDERR=${REDERR:-'2>'}
+
+REGRID_EXEC=${HOMEgfs}/sorc/gdas.cd/build/bin/regridStates.x
+
+export PGM=${REGRID_EXEC}
+export pgm=${PGM}
+
+NMEM_REGRID=${NMEM_REGRID:-1}
+CASE_IN=${CASE_IN:-${CASE_ENS}}
+LFHR=${LFHR:-6}
+
+# get resolutions
+CRES_IN="${CASE_IN:1}"
+LONB_CASE_IN=$((4*CRES_IN))
+LATB_CASE_IN=$((2*CRES_IN))
+
+CRES_OUT="${CASE_OUT:1}"
+ntiles=6
+
+APREFIX_ENS="enkfgdas.t${cyc}z."
+
+LSOIL_INCR=${LSOIL_INCR:-2}
+
+n_vars=$(( LSOIL_INCR*2 ))
+
+soil_incr_vars=""
+for vi in $( seq 1 "${LSOIL_INCR}" ); do
+    soil_incr_vars=${soil_incr_vars}'"soilt'${vi}'_inc"',
+done
+for vi in $( seq 1 "${LSOIL_INCR}" ); do
+    soil_incr_vars=${soil_incr_vars}'"slc'${vi}'_inc"',
+done
+
+cat << EOF > regrid.nml
+ &config
+  n_vars=${n_vars},
+  variable_list=${soil_incr_vars}
+  missing_value=0.,
+ /
+ &input
+  gridtype="gau_inc",
+  ires=${LONB_CASE_IN},
+  jres=${LATB_CASE_IN},
+  fname="enkfgdas.sfci.nc",
+  dir="./",
+  fname_coord="gaussian_scrip.nc",
+  dir_coord="./"
+/
+
+ &output
+  gridtype="fv3_rst",
+  ires=${CRES_OUT},
+  jres=${CRES_OUT},
```

Comment:
```suggestion
  ires=${CASE_OUT:1},
  jres=${CASE_OUT:1},
```

Reply:
@aerorahul Why would we truncate these?

Reply:
The suggestion was to use `CASE_OUT` truncated to the digits; i.e. C384 -> 384, instead of having to declare a separate variable just for that purpose. 
I am fine with rejecting the suggestion.

Reply:
Done.